data_safe_haven/commands/deploy_shm.py

```diff
@@ -50,17 +50,7 @@ def deploy_shm(
         )
         stack.add_option("azure-native:tenantId", config.azure.tenant_id, replace=False)
         # Add necessary secrets
-        stack.add_secret("password-domain-admin", password(20), replace=False)
-        stack.add_secret(
-            "password-domain-azure-ad-connect", password(20), replace=False
-        )
-        stack.add_secret(
-            "password-domain-computer-manager", password(20), replace=False
-        )
         stack.add_secret("password-domain-ldap-searcher", password(20), replace=False)
```

Comment:
Is this a credential Guacamole users to find users?
Are we not able to replace it here?

Reply:
It's the password for a user account on the domain controller, so both the SHM (to create it) and the SRE (to use it) need to know what it is.